assets/css/main.css

```diff
@@ -525,3 +648,17 @@ h1 a, h2 a, h3 a, h4 a {
   color: red;
   padding-left: .25em;
 }
+
+body.about .footer {
+  color: white !important;
+}
+
+/* Keyframes
+-------------------------------------------------- */
+@keyframes fadeIn {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+.fadeIn {
+  animation-name: fadeIn;
+}
```

Comment:
you can use animate.css for this -- just add class 'animated fadeIn' -- https://daneden.github.io/animate.css/